### PR TITLE
Allow configuring logrotate policy for cccp.log.

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

File: provisions/hosts.sample [hunk 1]
```diff
@@ -34,3 +34,5 @@ setup_nfs=False
 [jenkins_master:vars]
 # update as needed
 oc_slave=jenkins-slave
+logrotate_maxsize=100M
+logrotate_rotate=5
```
Comment (bamachrn) [inline provisions/hosts.sample]: @rtnpro why dont we keep it under all vars?

Comment (rtnpro) [inline provisions/hosts.sample]: @bamachrn I did not put the vars in ``all:vars`` just to show that these belong to ``jenkins/master``  role and are used there only. However, it does not stop anyone to put it in ``all:vars`` in the inventory file.

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
@bamachrn this is merge ready!

**Pull request merged**